### PR TITLE
bump dependencies

### PR DESCRIPTION
**Pull request by kimhanbeom**
1. Update giganto-client to 0.20.0
- Add more network data type. (`Bootp`, `Dhcp`)
- Update the protocol version of the REview, giganto.
   - Change `REVIEW_PROTOCOL_VERSION` to "0.38.0".
   - Change `PUBLISH_PROTOCOL_VERSION` to "0.21.0".
   - Change `INGEST_PROTOCOL_VERSION` to "0.21.0".

2. Update review-protocol version to 0.6.0
- Modify to use `ConnectionBuilder` to simplify the handling of connections with Central Manager.
- As `set_config` was removed from `review-protocol::request::handler`, Remove the `set_config` 
  related code from the crusher.

    
Close: #96
Close: #98

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -72,8 +70,14 @@ async fn main() -> Result<()> {
             )
         })?;
         let key = to_private_key(&key_pem).context("cannot read private key")?;
-        let ca_cert_paths = settings.ca_certs.clone();
-        let ca_certs = to_ca_certs(&ca_cert_paths).context("failed to read CA certificates")?;
+        let mut ca_certs_pem = Vec::new();
+        for ca_cert in settings.ca_certs {
+            let file = fs::read(&ca_cert).with_context(|| {
+                format!("failed to read ca certificate file: {}", ca_cert.display())
```
Comment (syncpark) [inline src/main.rs:76]: 여기도 밑의 메시지처럼 ca를 CA로 통일시키는게 좋겠습니다.

File: src/main.rs [hunk 2]
```diff
@@ -190,22 +196,18 @@ fn to_private_key(pem: &[u8]) -> Result<PrivateKeyDer<'static>> {
...
-                root_cert.add(cert_der).context("Failed to add root cert")?;
-            }
+    for ca_cert_pem in ca_certs_pem {
+        let root_certs: Vec<CertificateDer> = rustls_pemfile::certs(&mut &**ca_cert_pem)
+            .collect::<Result<_, _>>()
+            .context("invalid PEM-encoded certificate")?;
+        if let Some(cert) = root_certs.first() {
+            root_cert
+                .add(cert.to_owned())
+                .context("failed to add ca certificate")?;
```
Comment (syncpark) [inline src/main.rs:208]: 여기서도 ca -> CA로 통일이 필요합니다.

**Review by syncpark — CHANGES_REQUESTED** [review]

File: src/main.rs (hunk 1 above)
Comment (kimhanbeom) [reply src/main.rs]: done

File: src/main.rs (hunk 2 above)
Comment (kimhanbeom) [reply src/main.rs]: done

**Review by syncpark — APPROVED** [review]

**Pull request merged**